Directory.Build.targets

```diff
@@ -0,0 +1,31 @@
+<Project>
+    <Target
+            Name="PublishSamples"
+            Condition="'$(IsSample)' == 'true'">
```

Comment:
but we don't want this to fire for the actual library (which is why the Samples now have a D.B.props that adds this property).